WooCommerce/Classes/ViewRelated/Dashboard/Blaze/BlazeCampaignDashboardViewModel.swift

```diff
@@ -195,6 +199,17 @@ final class BlazeCampaignDashboardViewModel: ObservableObject {
             await reload()
         }
     }
+
+    func refreshSelectedCampaign() {
```

Comment:
Non-blocking suggestions: 
1. Consider adding a comment to explain how refreshing the campaign upon dismiss helps. 
2. We could also add a separate method `onDismiss` and call the `refreshSelectedCampaign` from there. This helps make the view dumb and keeps the reasoning inside the view model.

Reply:
Done in 92b055d225963e56038b0bc2655300af0bb57f25.